idemp-rest-test/testing/src/tests/windowAccess.test.ts

```diff
@@ -21,6 +21,7 @@ const windowUuid = {
 	transferInventory: 'd3c84cad-7306-464d-85da-7e629846f8c0',
 	visitsBills: 'a1f3e45c-4a6f-4c05-af26-517b8e9cbb77',
 	vitals: '53b4d743-c311-40e5-aa8e-c0880c42c1b1',
+	otcPharmacySales: '3a4ac3cd-9e1b-4a2c-82d3-78f698ec9e1f',
 } as const;
 
 test('admin role has correct access', async () => {

```

Comment:
Can we add tests for each of the roles?

Reply:
If we can also add checks for the other roles (to make sure the menus DON'T appear), I think that'll be good.

Reply:
Great